.github/workflows/docs.yml

```diff
@@ -19,7 +19,7 @@ jobs:
           with:
             timeout_minutes: 10
             max_attempts: 3
-            command: composer install
+            command: composer require --dev sami/sami
```

Comment:
Should we not do a `composer install` as well?

Reply:
a `require` does a `composer install`, so it would not do anything